### ISSUE TITLE
where to find libDetXR?

### ISSUE THREAD

**Issue by jreadey** [ganymede42/h5pyViewer#2]
I was trying to run this, but couldn't find the package for libDetXR.  Where do I get it?

**Comment by urquhart-usask** [comment]
Did you manage to solve this problem?

**Comment by jreadey** [comment]
No, I haven't done anything since submitting this issue last year.  It looks like there hasn't been any activity on the project either.  It looks promising based on the screen shots.

**Comment by urquhart-usask** [comment]
Thanks.

I am now using 

HdfView:
               https://support.hdfgroup.org/products/java/release/download.html

Seems to work; downloadable binary.

cheers,
Stephen

**Comment by vakkov** [comment]
@maheshpaliwal Do you, by any chance have the libDetXR file?
